app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def calculate_team_total_rating(players: list) -> int:
+    return sum([player.get_rating() for player in players])
+
+
+def elves_concert(elfs: list) -> None:
+    [elf.play_elf_song() for elf in elfs]
+
+
+def feast_of_the_dwarves(dwarfs: list) -> None:
+    [dwarf.eat_favourite_dish() for dwarf in dwarfs]
```

Comment:
add type annotation to list

Reply:
sorry, i don`t get you, can you give me more explanation

Reply:
list[Elf]